python/CMakeLists.txt

```diff
@@ -169,3 +182,25 @@ python_module(
   TARGET_NAME python-pipeline-description MODULE_FILES
   ${PYTHON_PIPELINEDESCRIPTION_FILES} MODULE_GENERATED_FILES
   revng/pipeline_description/_generated.py)
+
+#
+# Wheel generation
+#
+configure_file(
+  pyproject.toml "${CMAKE_BINARY_DIR}/${PYTHON_INSTALL_PATH}/pyproject.toml"
+  COPYONLY)
+
+add_custom_command(
+  OUTPUT "${CMAKE_BINARY_DIR}/revng-1.0.0-py3-none-any.whl"
+  COMMAND python -m build --no-isolation --wheel --outdir "${CMAKE_BINARY_DIR}"
+          &> /dev/null
+  DEPENDS "${CMAKE_BINARY_DIR}/${PYTHON_INSTALL_PATH}/pyproject.toml"
```

Comment:
As discussed, let's have two wheels. One internal and the other one public.

Reply:
Done